src/auth/password.ts

```diff
@@ -0,0 +1,24 @@
+import bcrypt from 'bcrypt'
+
+const saltRounds = 12
```

Comment:
This is a seguestion really.

Reply:
i thought about it, but didn't really do anything since we don't have a convention for this case since it's not in the server folder where we use the ConfigVar.js file. i feel like having a ConfigVar.js file in the auth folder might be a bit messy because all the files are so different and not that related. 

I also don't really know if it should be an enviroment variable since it's not something we are going to change often

Reply:
no, wait, i changed my mind, i'll add a pepper so i might as well add salt rounds as an enviroment variable. that way the seeding (not that it's important) can use the same amount of salt rounds